src/quad.rs

```diff
@@ -0,0 +1,102 @@
+use crate::terms::BlankNode;
+use crate::terms::Literal;
+use crate::terms::NamedNode;
+use std::fmt::{Display, Formatter};
+
+#[derive(PartialEq)]
+struct Quad {
```

Comment:
Should be marked as `pub`, along with its fields.

Reply:
I see that `NamedNode`'s field is marked as `pub(crate)`, so that would likely be used here as well for `Quad`'s fields.

Reply:
actually, when I changed the strategy to use `enum` for quad, that crate-level visibility was no longer necessary, so I removed it.